test/engine/test_apps.js

```diff
@@ -32,6 +32,24 @@ async function collectOutputs(app) {
     return into;
 }
 
+async function testActivityMonitor(engine) {
+    const app = await engine.createApp('timer(base=$now,interval=10s) => @org.thingpedia.builtin.test.get_data(count=2, size=10byte) => notify;',
+            { icon: 'org.foo', uniqueId: 'uuid-timer-foo', name: 'some app', description: 'some app description' });
+    let idle = false;
+    engine.activityMonitor.on('idle', () => idle = true);
+
+    // inactivity timers should not start when there are apps
+    await delay(2000);
+    assert(engine.apps.hasApp(app.uniqueId));
+    assert(idle === false);
```

Comment:
`assert.strictEqual(idle, false)` will give you better errors when the assertion fails.